includes/forms/class-wp-job-manager-form-submit-job.php

```diff
@@ -173,7 +173,7 @@ public function init_fields() {
 					'label'       => __( 'Job type', 'wp-job-manager' ),
 					'type'        => $job_type,
 					'required'    => true,
-					'placeholder' => '',
+					'placeholder' => __( 'Choose job type&hellip;', 'wp-job-manager' ),
```

Comment:
That hellip is to override this one:
https://github.com/Automattic/WP-Job-Manager/blob/v1.14.0/wp-job-manager-functions.php#L442

Which also using hellip (so, it's consistent)
Do you want me to change it to something else too?